src/Components/Chatroom.js

```diff
@@ -0,0 +1,89 @@
+import { useEffect, useRef, useState } from "react";
+import firebase from "firebase/app";
+import "firebase/firestore";
+import { formatRelative } from "date-fns";
+import Image from "next/image";
+
+export default function ChatRoom(props) {
+  const db = props.db;
+  const dummySpace = useRef();
+  const { uid, displayName, photoURL } = props.user;
+  const [Chat12, setMessages] = useState([]);
```

Comment:
try good naming convention. What is Chat12 here?

Reply:
ok